perllib/FixMyStreet/App/Controller/Waste/Bulky.pm

```diff
@@ -328,7 +328,27 @@ sub process_bulky_amend : Private {
 
     my $p = $c->stash->{amending_booking};
 
-    if ($c->cobrand->bulky_cancel_by_update) {
+    # In this case we don't want to go through the process of cancelling the original
+    # report as we don't want to risk losing the booking slot
+    if ( $c->cobrand->call_hook('update_report_for_same_day_amend', $p, $data) ) {
+        $p->create_related( moderation_original_data => {
+            title => $p->title,
+            detail => $p->detail,
+            photo => $p->photo,
+            anonymous => $p->anonymous,
+            category => $p->category,
+            extra => $p->extra,
+        });
+        $c->forward( '/admin/log_edit', [ $c->cobrand->body->comment_user->id, 'problem', 'edit' ] );
+        $p->photo('');
```

Comment:
Is this line needed? Looks like amend_extra_data always sets `$p->photo` anyway.